src/configuration/app-containers.md

```diff
@@ -211,37 +212,37 @@ web:
 #### Upstream
 
 `upstream` is an optional key that describes how your application listens to
-requests, and what protocol it speaks.
+requests and what protocol it speaks.
 
 The following subkeys can be defined:
 * `socket_family`:
     Default: `tcp`. Describes whether your application will listen on a
-    Unix socket or a TCP socket. Can be either `tcp` or `unix`.
+    Unix socket or a TCP socket and can be either `tcp` or `unix`.
```

Comment:
Hm. I agree we shouldn't have a sentence fragment here, but the alternative feels off to me, too.  I'm open to suggestions here.